network/client.go

```diff
@@ -61,13 +61,6 @@ func NewClient(conn net.Conn, endpoint *Endpoint, ip string) *Client {
 	}
 }
 
-// recordRequest logs a request as an id/method pair.
-// func (c *Client) recordRequest(id uint64, method string) {
```

Comment:
let's leave this commented out for now, there might be some stratum calls yet to be implemented that will use this.